src/ducks/settings/Configuration.jsx

```diff
@@ -82,13 +108,54 @@ const editModalProps = {
         }
       }
     },
-    fieldOrder: singleValueFieldOrder,
+    fieldOrder: ['value'],
     fieldLabels: {
       value: t('Notifications.when_late_health_reimbursement.fieldLabels.value')
     }
   })
 }
 
+const getTransactionGreaterDescriptionKey = props => {
+  if (props.doc && props.doc.accountOrGroup) {
+    return 'Notifications.if_transaction_greater.descriptionWithAccountGroup'
+  } else {
+    return 'Notifications.if_transaction_greater.description'
+  }
+}
```

Comment:
Since it only uses `props.doc.accountOrGroup`, wouldn't it be better to directly pass `accountOrGroup` to this function? When I see the `props` name, I think the function is a react component. But here it's a regular getter function.

Reply:
Good point, I thought it was more flexible to use directly props, but doc is sufficent.

Reply:
I actually need the full props to have access to getAccountOrGroupLabel from the props.

Reply:
More explanation here : https://github.com/cozy/cozy-banks/pull/1711/commits/c104d08c8d29a3a8e0a04f8fe5fefc9942b9625d